march_simulation/launch/march_world.launch

```diff
@@ -10,9 +10,9 @@
   <!-- Load the URDF into the ROS Parameter Server -->
   <!-- Override effort values as Gazebo uses different units than the actual hardware. -->
   <!-- 40 seems to work well, I do not know why. -->
```

Comment:
Outdated comment

Reply:
Fix in https://github.com/project-march/simulation/pull/26. Thanks for the spot :+1: